pkg/miniogw/gateway-storj.go

```diff
@@ -247,6 +247,41 @@ func (s *storjObjects) MakeBucketWithLocation(ctx context.Context,
 	return err
 }
 
+func (s *storjObjects) CopyObject(ctx context.Context, srcBucket, srcObject, destBucket,
+	destObject string, srcInfo minio.ObjectInfo) (objInfo minio.ObjectInfo, err error) {
+	defer mon.Task()(&ctx)(&err)
+	o, err := s.storj.bs.GetObjectStore(ctx, srcBucket)
```

Comment:
I'd avoid duplicating the code from GetObject here. Now we will have to maintain in it two places. Could you extract the common part in a helper method (like `getObject()`).